libraries/chain/smt_objects/nai_pool.cpp

```diff
@@ -113,8 +113,12 @@ void remove_from_nai_pool( database &db, const asset_symbol_type& a )
    if ( it != end )
    {
       auto index = std::distance( nais.begin(), it );
+
+      wlog( "Using NAI ${n}", ("n",a) );
```

Comment:
We shouldn't have these warnings checked in to `master`, it seems they are there for debugging.